app/controllers/api/v1/project_roles_controller.rb

```diff
@@ -10,9 +10,13 @@ def resource_name
   end
 
   def update
-    super do
-      UserAddedToProjectMailerWorker.perform_async(api_user.id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
-    end
+    UserAddedToProjectMailerWorker.perform_async(acl_user_id, controlled_resource.resource.id, roles) if new_roles_present?(roles)
```

Comment:
Second, also no. There is no linked user in the params of a call to `/project_roles/:id`. PFE makes a few calls after the event, culminating in a final PUT to the above route with a body containing only an array of roles. Hence the roundabout way of getting at the user ID.